plugins/state_monitor.py

```diff
@@ -0,0 +1,185 @@
+"""
+Plugin to monitor state variables from the charge controller.
+"""
+
+# Standard library imports
+
+# Local imports
+import brokkr.pipeline.base
+import brokkr.pipeline.decode
+
+from notifiers.slack import SlackSender
+
+
+class StateMonitor(brokkr.pipeline.base.OutputStep):
+    """Handle notifications for changes in state variables."""
+
+    def __init__(self,
+                 method=None,
+                 power_delim=1,
+                 slack_channel=None,
+                 slack_key_file=None,
+                 **output_step_kwargs,
+                 ):
```

Comment:
No problem. But, I prefer to have self on the same line as init, since you always need self. It just makes more sense to me. But, that's my hang up :-)